src/Client/index.ts

```diff
@@ -77,17 +77,15 @@ export class Client {
     };
 
     if (options.dnsDiscover) {
-      const { address } = options.hosts[0];
```

Comment:
**COMMENT:** Minor but wouldn't it better to still keep the variable if we're using it in more than one place?